src/core/context/CommentActivityContext.tsx

```diff
@@ -0,0 +1,10 @@
+import { createContext, useContext } from 'react';
```

Comment:
This way of importing/using react module elements seems inconsistent with the one used on other places in the code i.e.[React.useMemo instead of just useMemo](https://github.com/makerdao-ses/ecosystem-dashboard/pull/496/files#diff-6177c1c9dbecf396a26ad5b3eaf7180aa173734b7b8492b117c97feed111376fL38). Since React object is global it'll make no difference technically(IMHO) but maybe it's worth changing it in one of the places for the sake of consistency

Reply:
You're right, we should ensure consistency here, we should be adding a eslint rule for that. The way is done here should be the one used in the entire project since `React.useXXX` is more verbose and just using `useXXX` is enough as this is the convention